GEM_Workshop/LC_CMS_Workshop/notebook/config_files/global_config.json

```diff
@@ -0,0 +1,46 @@
+{
+  "variables": {
+    "aoi_filename": "paris-aoi.geojson",
+    "project_dir": "/gem-workshop/paris_run/2020"
+//    "project_dir": "/Users/dhiman/Stories/gem-workshop/orleans_workdir"
```

Comment:
there are many commented-out lines in the configs, is this intended?

Reply:
fixed